tests/main/interfaces-openvswitch/task.yaml

```diff
@@ -3,6 +3,8 @@ summary: Ensure that the openvswitch interface works.
 systems:
     - -ubuntu-core-*
 
+manual: true
```

Comment:
A small note and a link to the debian bugreport would be great. This way we will remember what the issue was down the line :)